code-dot-org/code-dot-org
# Course catalog shows the recommended versions.

# Description
When international students and teachers would visit
https://code.org/courses , they were shown a list of courses hardcoded
to the 2017 versions. This change makes it so the catalog instead
displays the recommended version of the course in the user's language
instead. For example, the 2018 version of Course A has been fully
translated to Italian, so the 2018 version of Course A will be displayed
on the course catalog instead. This way students and teachers can easily
find the latest, recommended content.
* Updated course catalog to display the recommended course version for international users.
* Copy/Pasted course preview images from 2017 -> 2018,2019.

## Links
- [jira](https://codedotorg.atlassian.net/browse/FND-173)

## Screenshots
![image](https://user-images.githubusercontent.com/1372238/72179403-b8888d80-33dc-11ea-99a9-f2dce5632a30.png)

## Testing story
* `./bin/dashboard-server`
  * Visit http://localhost-studio.code.org:3000/courses/lang/it-IT
  * Verify the 2018 courses are shown

# Reviewer Checklist:
